lib/types.ts

```diff
@@ -29,3 +30,18 @@ export interface TokenInfo {
   description?: string
   logoURI?: string
 }
+
+export interface NFTCollectionInfo {
+  id: string
+  name: string
+  symbol: string
+  description?: string
+}
+
+// JSON Schema for the NFT metadata, which is pointed to by
+// the `uri` field of the NFT
+export interface NFTMetadata {
+  name: string
+  description: string
+  image: string
+}
```

Comment:
I don't think this one is needed, we won't add each nft in our list. Or maybe it's just to have the schema in the TS lib?

Reply:
Yeah, it is to just have the schema in the lib. But maybe we can have it in web3 lib instead :thinking:

Reply:
Maybe it is better to stay here since this lib is light weight and if someone wants to use this definition, it won't have to depend on the entire web3 lib. WDYT?

Reply:
>  it won't have to depend on the entire web3 lib

Good point, fully agree. Let's keep it there